packages/teleport/src/services/joinToken/makeJoinToken.ts

```diff
@@ -14,12 +14,9 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-import { at } from 'lodash';
-import { NodeToken } from './types';
-
-export default function makeToken(json): NodeToken {
-  const [id, expiry] = at(json, ['id', 'expiry']);
+import { JoinToken } from './types';
 
+export default function makeToken({ id, expiry }): JoinToken {
```

Comment:
@hatched actually made me realize something we do at other places and forgot in other places, just as a safeguard if object happens to be `undefined`: 

[checks for undefined
](https://github.com/gravitational/webapps/blob/master/packages/teleport/src/services/apps/makeApps.ts#L21)[tests for undefined](https://github.com/gravitational/webapps/blob/master/packages/teleport/src/services/apps/apps.test.ts#L46) (or null same thing in this case)
```suggestion
export default function makeToken({ id, expiry } = {}): JoinToken {
```

Reply:
If you wanted, to fix that error you can provide default values in the functions call signature...

```js
function foo ({bar, baz} = {bar: 'zero', baz: 0}) {
  console.log(bar, baz);
}
foo();
// --> zero 0
```

Reply:
yeah normally we check if something is undefined before giving it to something that may error out, the way I would do it is:

```
function foo ({id, expiry} = {}) {
  return {
      id,
      expiry: expiry ? new Date(expiry) : null, // then let the root caller handle the response values
  }
}
```

the lodash `at`, i think did something similar (which explains the possible usage of it in the past)

Reply:
Thanks, incorporated these feedbacks on https://github.com/gravitational/webapps/pull/672/commits/9adcc81bc66a3e2e4fe036d1bbfa4e4e86bd07f9